### PR TITLE
New version: Scala.Scala.2 version 2.13.11.400

### PR DESCRIPTION
**Pull request by superusercode**
###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/109483&drop=dogfoodAlpha

### CODE REVIEW

**Comment by stephengillie** [comment]
This package installs website links, and `JAR`s. Should a `JRE` be a dependency?

File: manifests/s/Scala/Scala/2/2.13.11.400/Scala.Scala.2.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: uninstallPrevious
+ReleaseDate: 2023-06-02
+AppsAndFeaturesEntries:
+- UpgradeCode: '{6061C134-67C7-4FB2-AFF5-32B01A186968}'
+Installers:
```
Comment (stephengillie) [inline manifests/s/Scala/Scala/2/2.13.11.400/Scala.Scala.2.installer.yaml:19]:
```suggestion
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Oracle.JDK.17
Installers:
```

**Comment by ItzLevvie** [comment]
[Policy] Blocking-Issue

**Comment by Trenly** [comment]
Dependencies are now supported in the 1.6 client. If you could please submit a new manifest for the latest version of the package, it would be greatly appreciated!

**Pull request closed without merging**